index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.4.1/dist/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
+    <link rel="stylesheet" href="index.css">
+    <title>Document</title>
```

Comment:
Always to include a precise title for every html document. This helps user have a sense of what content is contained  on the browser tab.
Happy Coding

Reply:
- Great job for describing your comment with reason, 👍👍however, it is better to start with politely phrased words, as suggestions instead of orders. For example:
   * This is a polite suggestion: *"I think it would be better to use title for every..."*
   * This reads as an order: *"Always to include a precise title"*👍👌
- [Optional] Kindly use a couple of emojis and/or gifs in the review.
They make the code review more human-friendly 👽🤖.